test/powershell/Modules/Microsoft.PowerShell.Diagnostics/Import-Counter.Tests.ps1

```diff
@@ -212,22 +212,15 @@ function RunExpectedFailureTest($testCase)
         else
         {
             # Here we expect and want the command to fail
-            try
+            $sb = [ScriptBlock]::Create($cmd)
+            $e = { &$sb } | Should -Throw
+            if ($testCase.ExpectedErrorId)
```

Comment:
Perhaps we should always check `ExpectedErrorId` and change line 216 above.  Find the test case that doesn't have `ExpectedErrorId` defined and put in the actual errorid that occurs.

Reply:
ExpectedErrorId seems to always be there, so I removed the if statement.